sdcm/stress_thread.py

```diff
@@ -136,10 +136,27 @@ def create_stress_cmd(self, node, loader_idx, keyspace_idx):
 
     @staticmethod
     def _add_errors_option(stress_cmd: str, to_add: list):
-        to_add = (' '.join(to_add))
-        if ' -errors' not in stress_cmd:
-            return stress_cmd + ' -errors ' + to_add
-        return re.sub('-errors([ ]+[^-][a-zA-Z0-9-]+)+([ ]* -[a-z]+|[ ]*)', '-errors\\1 ' + to_add + '\\2', stress_cmd)
+        """
+        Add suboption to -errors option, if such suboption is there, does not add or change it
+        """
+        options = stress_cmd.split(' -')
+        to_add = list(to_add)
+        error_option = None
+        for option in options:
+            if option.startswith('errors '):
+                error_option = option
+                break
+        if error_option is None:
+            return stress_cmd + ' -errors ' + ' '.join(to_add)
+        to_add_heads = [suboption.split('=')[0] for suboption in to_add]
+        suboption_heads = [suboption.split('=')[0] for suboption in error_option.split(' ')]
```

Comment:
Nit: IMO it would be more readable to use `current_heads` instead of `suboption_heads`

Because `to_add_heads` is also a list of suboption heads. What really differs these two variables is that one of them describes the suboptions we want to add, the other - suboptions that we already have.

Reply:
Also: suppose that
```python
error_option = 'errors a b c x=1 y=2'
```
then
```python
error_option.split(' ') = ['errors', 'a', 'b', 'c', 'x=1', 'y=2']
```
since we only want to take the heads of suboptions, maybe we should instead use
```
error_option.split(' ')[1:] = ['a', 'b', 'c', 'x=1', 'y=2']
```

Reply:
Thanks, fixed.